### PR TITLE
update/remove dependencies

### PR DESCRIPTION
**Pull request by gonpombo8**
# What? 

...

# Why? 

...

### CODE REVIEW

File: test/helpers/sandbox.ts [hunk 1]
```diff
@@ -12,13 +11,13 @@ export default function sandbox(fn: CallbackFn) {
...
     const done = () => {
-      rimraf(dir, resolve)
+      fs.remove(dir, resolve)
     }
 
     try {
       fn(dir, done)
     } catch (e) {
-      rimraf(dir, () => reject(e))
+      fs.remove(dir, () => reject(e))
```
Comment (menduz) [inline test/helpers/sandbox.ts:20]: fs.remove works for recursive directories???

Comment (gonpombo8) [reply test/helpers/sandbox.ts:20]: i just try it and yes, it works :)

Comment (menduz) [reply test/helpers/sandbox.ts:20]: What ensures it will finish? Maybe adding an await?

File: test/unit/lib/Project.test.ts [hunk 2]
```diff
@@ -142,12 +142,14 @@ test('Unit - Project.validateNewProject() - should pass if the working directory
   t.pass()
 })
 
-test('Unit - Project.validateNewProject() - should fail if the working directory contains a scene.json file', async (t) => {
+test.only('Unit - Project.validateNewProject() - should fail if the working directory contains a scene.json file', async (t) => {
```
Comment (leanmendoza) [inline test/unit/lib/Project.test.ts:145]: maybe remove the `.only` ?

File: test/helpers/sandbox.ts (hunk 1 above)
Comment (leanmendoza) [reply test/helpers/sandbox.ts:20]: if it fails or not, it's calling reject in the callback.

**Review by leanmendoza — APPROVED** [review]

File: test/unit/lib/Project.test.ts (hunk 2 above)
Comment (gonpombo8) [reply test/unit/lib/Project.test.ts]: 🫠

**Pull request merged**